aptos-move/aptos-vm-types/src/storage.rs

```diff
@@ -125,8 +125,8 @@ impl StoragePricingV2 {
         }
     }
 
-    fn write_op_size(&self, key: &StateKey, value: &[u8]) -> NumBytes {
-        let value_size = NumBytes::new(value.len() as u64);
+    fn write_op_size(&self, key: &StateKey, value_size: u64) -> NumBytes {
```

Comment:
maybe value size? as WriteOpSize is now a different type

Reply:
oh I addressed your comment, but then thought comment referred to the name of the function, not the argument :D